### ISSUE TITLE
Can't include react-infinite with npm

### ISSUE THREAD

**Issue by travisperson** [seatgeek/react-infinite#25]
Trying to include react-infinite in a project that uses browserify results in an error. I believe this is because the package.json file points to an already browserified file.

```
Error: module "./computers/constant_infinite_computer.js" not found from "/home/travis/tmp/crossfilter/node_modules/react-infinite/dist/react-infinite.js"
    at notFound (/home/travis/tmp/crossfilter/node_modules/gulp-browserify/node_modules/browserify/index.js:803:15)
    at /home/travis/tmp/crossfilter/node_modules/gulp-browserify/node_modules/browserify/index.js:754:23
    at /home/travis/tmp/crossfilter/node_modules/gulp-browserify/node_modules/browserify/node_modules/browser-resolve/index.js:185:24
    at /home/travis/tmp/crossfilter/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:36:22
    at load (/home/travis/tmp/crossfilter/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:54:43)
    at /home/travis/tmp/crossfilter/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:60:22
    at /home/travis/tmp/crossfilter/node_modules/gulp-browserify/node_modules/browserify/node_modules/resolve/lib/async.js:16:47
    at FSReqWrap.oncomplete (fs.js:99:15)

```

**Comment by pviolette3** [comment]
I also have this problem.

**Comment by garetht** [comment]
@travisperson @pviolette3 fixed in `0.3.4`. Browserify now draws from source, and Reactify is required to transform the source.
